cilium-health/launch/endpoint.go

```diff
@@ -41,6 +41,7 @@ import (
 	"github.com/cilium/cilium/pkg/option"
 	"github.com/cilium/cilium/pkg/pidfile"
 
+	"github.com/cilium/cilium/pkg/metrics"
```

Comment:
Move this import to the import group above.

Reply:
sure